CanalTP/Hermod
# mailer_host added in env-map

#### What I did

I added `MAILER_HOST` in env-map to initialized `mailer_host` in parameters.yml

#### How I did it

I added it in `composer.json`, `env-map` section

#### How to verify it

You should set `mailer_host` value in `parameters.yml` when you do `composer install` with `MAILER_HOST` in env
